feedback.md

```diff
@@ -1,31 +1,61 @@
 # Media Ranker
 ## What We're Looking For
 
-Feature | Feedback
----     | ---
-**Core Requirements** |
-Git hygiene |
-Comprehension questions	|
-**General** |
-Rails fundamentals (RESTful routing, use of named paths) |
-Views are well-organized (DRY, use of semantic HTML, use of partials) |
-Errors are reported to the user |
-Business logic lives in the models |
-Models are thoroughly tested, including relations, validations and any custom logic |
-Controllers are thoroughly tested, including the login/logout process and multi-step workflows like voting for a work |
-**Wave 1 - Media** |
-Splash page shows the three media categories |
-Basic CRUD operations on media are present and functional |
-**Wave 2 - Users and Votes** |
-Users can log in and log out |
-The ID of the current user is stored in the session |
-A user cannot vote for the same media more than once |
-All media lists are ordered by vote count |
-Splash page contains a media spotlight |
-**Wave 3 - Users and Votes** |
-Media pages contain lists of voting users |
-Individual user pages and the user list are present |
-**Optional - Styling** |
-Bootstrap is used appropriately |
-Look and feel is similar to the original |
-**Overall** |
+<!-- Reviewer Instructions: Limit your in-line comments on this one. Also, the manual testing is huge, but covers eeeeverything, so if there's anything to comment on as constructive feedback, the manual testing should point you right to it with any line that has the answer "no" -->
+
+### Manual Testing
+
+Workflow | yes / no
+--- | ---
+Deployed to Heroku | 
+Look and feel is similar to the original (consider styling and layout) | 
```

Comment:
This requirement is also in the Optional - Styling section of the previous table.

Reply:
nevermind, that table is removed.